support/ebpf/frametypes.h

```diff
@@ -38,9 +34,10 @@
 // Indicates a BEAM frame
 #define FRAME_MARKER_BEAM    0xC
 
-// Indicates a frame containing information about a critical unwinding error
-// that caused further unwinding to be aborted.
-#define FRAME_MARKER_ABORT (0x7F | FRAME_MARKER_ERROR_BIT)
+// Frame flags
+#define FRAME_FLAG_ERROR          1
+#define FRAME_FLAG_RETURN_ADDRESS 2
+#define FRAME_FLAG_PID_SPECIFIC   4
```

Comment:
nit: this feels easier to read:
```suggestion
#define FRAME_FLAG_ERROR          1<<0
#define FRAME_FLAG_RETURN_ADDRESS 1<<1
#define FRAME_FLAG_PID_SPECIFIC   1<<2
```